qa/rpc-tests/mempool_limit.py

```diff
@@ -104,6 +51,7 @@ def run_test(self):
         txFS = self.nodes[0].signrawtransaction(tx, None, None, "NONE|FORKID")
         txid0 = self.nodes[0].sendrawtransaction(txFS['hex'])
         self.nodes[0].lockunspent(True, [us0])
+        time.sleep(3)
```

Comment:
why?

Reply:
Good catch, this was not necessary.

However, in more thorough testing I found the existing sleep had to be increased to 9 to ensure the test would succeed 30 times in a row.